packages/components/src/highlight-cards/annual-highlight-cards.tsx

```diff
@@ -27,9 +35,12 @@ export default function AnnualHighlightCards( {
 
 	const header = (
 		<h1 className="highlight-cards-heading">
-			{ Number.isFinite( year )
-				? translate( '%(year)s in review', { args: { year } } )
-				: translate( 'Year in review' ) }
+			<span>
+				{ Number.isFinite( year )
```

Comment:
Not related to the PR, but `Number.isFinite( year )`  doesn't look right, does it?

Reply:
I think it was added to ensure that we deal with a number or a proper value after some calculations. Would you use something else?

Reply:
IIRC it was to check whether `year` was a valid number.